swarm/network/kademlia.go

```diff
@@ -779,6 +785,9 @@ type Health struct {
 func (k *Kademlia) Healthy(pp *PeerPot) *Health {
 	k.lock.RLock()
 	defer k.lock.RUnlock()
+	if len(pp.NNSet) < k.MinProxBinSize {
```

Comment:
I see that this is required for discovery tests, but are we sure that this panic will not make problems in production?

Reply:
this function is only used in tests.  i should remove panic though and uncomment asserts